front/app/hooks/fixtures/pages.ts

```diff
@@ -61,7 +61,7 @@ const PAGES: IPageData[] = [
age including profanity</li>\n    <li class="ql-align-justify">Marketing or advertising material</li>\n    <li class="ql-align-justify">Personal information about other individuals</li>\n    <li class="ql-align-justify">Copyrighted material such as images</li>\n    </ul><p class="ql-align-justify">Any posts or comments that do not adhere to these guidelines may be edited or deleted, or in extreme instances, prevented from being posted. Please review our Terms &amp; Conditions for more information. </p><h4 class="ql-align-justify">How do I report an offensive post?</h4><p class="ql-align-justify">Click on the three small dots on the top right corner of the post to report it. Please indicate the reason you are reporting the content - this will send a notification for the post to be reviewed.</p><h4 class="ql-align-justify">Why am I having difficulty signing up or logging in?</h4><p class="ql-align-justify">If you’re unable to sign up, it may be because you’ve already registered with the same email address or previously received an invitation to join the platform. If you see a message that your email address is taken by a pending invite, please check your email and follow the link in the invitation to register your account. If you can’t find the invite, try looking in your spam box or request a new invitation. </p><h4 class="ql-align-justify">How do I reset my password?</h4><p class="ql-align-justify">You can always request to reset your password from the log in page. If you reset your password, a link will be sent to your email address - follow the instructions to change your password. For security reasons, the link is valid for one hour. If the link has expired, you can simply click on ‘Forgot password?’ in the log in page again to generate a new reset link.</p><p class="ql-align-justify">If you didn’t receive the email, check to make sure that it’s not in your spam and that you’re sending the link to the same email address that you signed up with.</p><h4 class="ql-align-justify">What will you do with my data?</h4><p class="ql-align-justify">Review our privacy policy for more information on data collection and usage. In short, we will never use your data for advertising, marketing or other types of spam.</p><h4 class="ql-align-justify">What emails will I receive?</h4><p class="ql-align-justify">You may receive emails related to your account and your activity on the platform, for instance, when someone responds to one of your posts. You can control which emails you receive in your Settings. There is a section called “Notifications” where you can turn email notifications on and off.</p><h4 class="ql-align-justify">How do I delete my account?</h4><p class="ql-align-justify">In your Settings there is the option to delete your account. Input that you’ve posted will remain published, but your name will be removed from the post and replaced with ‘unknown author.’ If you’d like to delete all of your input, please do so before deleting your account. </p>',
```

Comment:
Why did you pick h4? The title of the page ("FAQ") is an h1, so the logical next one, and the one that the editor of this page uses (go to e.g. https://demo.stg.citizenlab.co/en/admin/pages to edit the FAQ page) is h2.

We should also remove the \n in the li. You can run the HTML through https://validator.w3.org/#validate_by_input to see if it's valid.

(This means these changes are also needed in Crowdin.)